### PR TITLE
archiving events + home remake

### PR DESCRIPTION
**Pull request by JakubC-projects**

### CODE REVIEW

File: client/src/views/Home.vue [hunk 1]
```diff
@@ -42,31 +66,59 @@ export default defineComponent({
...
+        @apply gap-6;
+        @apply justify-items-stretch;
+    }
     .banner-text{
         height: 190px;
     }
-    @media (min-width: 640px) {
-    .banner-text{
-      height: 310px;
+    @media (min-width: 720px) {
```
Comment (StevenMalaihollo) [inline client/src/views/Home.vue:114]: These breakpoints are pretty random. I believe tailwind and the universal breakpoints for md or lg screens are 768px and 992px respectively

File: server/src/domain/Poll.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export enum PollActiveStatus {
     'Not Started' = 'not_started',
     'Live' = 'live',
-    'Finished' = 'finished'
```
Comment (StevenMalaihollo) [inline server/src/domain/Poll.ts:4]: Missing archived right

File: client/src/views/Home.vue [hunk 3]
```diff
@@ -42,31 +66,59 @@ export default defineComponent({
...
     },
     methods: {
         goToCreate() {
             this.$router.push({ path: '/create' })
         },
-        async loadMeetings(){
+        async loadPollingEvents(){
             const roleIds = this.$user.roles.map((r: any) => r.id)
-            this.pollingEvents = await this.$client.service('polling-event').find({})
+            const allEvents = await this.$client.service('polling-event').find({})
```
Comment (StevenMalaihollo) [inline client/src/views/Home.vue:88]: unused roleIds var

File: server/src/domain/Poll.ts (hunk 2 above)
Comment (JakubC-projects) [reply server/src/domain/Poll.ts:4]: The poll doesn't get archived, It's the polling event

File: client/src/views/Home.vue (hunk 1 above)
Comment (JakubC-projects) [reply client/src/views/Home.vue]: fixed that 720, the 640 is actually the universal small screen breakpoint

**Review by StevenMalaihollo — APPROVED** [review]

**Pull request merged**